src/main/scala/temple/test/internal/CRUDServiceTest.scala

```diff
@@ -24,58 +27,56 @@ class CRUDServiceTest(
       test.validateResponseBody(requestBody.asObject, createJSON, service.attributes)
     }
 
-  private def testReadEndpoint(accessToken: String): Unit =
+  private def testReadEndpoint(): Unit =
     testEndpoint("read") { test =>
+      val accessToken    = newToken()
       val createResponse = ServiceTestUtils.create(test, name, allServices, baseURL, accessToken)
       val getJSON = ServiceTestUtils
         .getRequest(
           test,
-          s"http://$baseURL/api/${StringUtils.kebabCase(name)}/${createResponse.id}",
-          createResponse.accessToken,
+          s"http://$baseURL/api/${StringUtils.kebabCase(name)}/${createResponse}",
```

Comment:
What was the reason for stripping the `http://` from the `baseURL` in performSetup?

Reply:
That's for providing the host when configuring Kong. https://docs.konghq.com/2.0.x/admin-api/#add-route